deps/scale.ts

```diff
@@ -1 +1 @@
-export * from "https://deno.land/x/scale@v0.4.0/mod.ts";
+export * from "https://raw.githubusercontent.com/paritytech/parity-scale-codec-ts/3b02d9e1e15a1fdfef0fdc8e62cd746afbc534e5/mod.ts";
```

Comment:
Waiting on paritytech/parity-scale-codec-ts#79
```suggestion
export * from "https://deno.land/x/scale@v0.4.2/mod.ts";
```